src/main/resources/templates/edit.html

```diff
@@ -53,7 +59,8 @@ <h2 class="mb-4 text-center">Valet Parking Edit Tracker</h2>
 
       <div class="input-group input-group-sm mb-3">
         <span class="input-group-text" id="notes">Notes</span>
-        <textarea type="text" class="form-control" rows="3" cols="40" maxlength="200" placeholder="Enter up to 150 characters" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="notes" th:field="*{vehicle.notes}"></textarea>
+        <textarea type="text" class="form-control" rows="3" cols="40" maxlength="200" placeholder="Enter up to 150 characters" 
+        aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="notes" th:field="*{vehicle.notes}"></textarea>
       </div>
 
```

Comment:
Added extra lines for the input field so it can be read all on one screen and added _required_ to Ticket Ic